### ISSUE TITLE
When multiple classes are present in the same file, only the last one is checked

### ISSUE THREAD

**Issue by georgewfraser** [uber/NullAway#388]
For example:

```java
package com.fivetran.demo;

import javax.annotation.Nullable;

class First {
    public void bad(@Nullable String foo) {
        foo.equals("foo");
    }
}

class Second {
    public void ok() {}
}
```

Foo contains an error, but no errors are detected when I build with Bazel. When I reverse the order of these classes, the error is found:

```
ERROR: /Users/george/Documents/engineering/nullable_demo/BUILD:1:1: Building nullable_demo/main.jar (1 source file) failed (Exit 1)
nullable_demo/src/Demo.java:13: error: [NullAway] dereferenced expression foo is @Nullable
        foo.equals("foo");
           ^
    (see http://t.uber.com/nullaway )
Target //nullable_demo:main failed to build
```

v0.7.9

**Comment by lazaroclapp** [comment]
Can't repro this on NullAway master.

I added it to `sample/src/main/java/com/uber/mylib/Foo.java`, changed package and got:

```
error: [NullAway] dereferenced expression foo is @Nullable
        foo.equals("foo");
           ^
    (see http://t.uber.com/nullaway )
1 error
```

We probably should cut a new version, since 0.7.9 is from November, but there is nothing since then that affects how classes are matched...

I can think of three things:

1. What's your EP version? Is this using the EP javac? What's the version on that? 
2. Are you building with `-XDcompilePolicy=simple`? (see https://github.com/google/error-prone/issues/781)
3. Some bazel quirk I don't know about...

**Comment by msridhar** [comment]
Closing this out as we couldn't repro, but re-open if it's still a problem
